src/main/java/net/syscon/elite/service/support/StringWithAbbreviationsProcessor.java

```diff
@@ -0,0 +1,84 @@
+package net.syscon.elite.service.support;
+
+import org.apache.commons.text.WordUtils;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Utility class containing methods for processing of strings which contain abbreviations.
+ */
+public class StringWithAbbreviationsProcessor {
+    /**
+     *
+     * @param string string to convert
+     * @return new location with correct titlecase
+     *
+     */
+    public static String format(final String string) {
+        // Handle the possibility of the userDescription being empty
+        if (string == null) {
+            return null;
+        }
+        var description = WordUtils.capitalizeFully(string);
+        // Using word boundaries to find the right string ensures we catch the strings
+        // wherever they appear in the description, while also avoiding replacing
+        // the letter sequence should it appear in the middle of a word
+        // e.g. this will not match 'mosaic' even though AIC is one of the abbreviations
+        Pattern pattern = Pattern.compile("\\b(" + String.join("|", ABBREVIATIONS) + ")\\b", Pattern.CASE_INSENSITIVE);
```

Comment:
Generally good practice to extract out Patterns as constants where possible. 
They are generally quite resource intensive to create, not that it will matter much in this code path - but its a bit of a convention if possible